X_mix-tasks.md

```diff
@@ -226,6 +231,39 @@ Compiled web/models/post.ex
 (stdlib) erl_eval.erl:657: :erl_eval.do_apply/6
 ```
 
+If we don't want to create a model for our resource we can use the `--no-model` flag.
+
+```console
+$ mix phoenix.gen.html Post posts body:string word_count:integer --no-model
+* creating web/controllers/post_controller.ex
+* creating web/templates/post/edit.html.eex
+* creating web/templates/post/form.html.eex
+* creating web/templates/post/index.html.eex
+* creating web/templates/post/new.html.eex
+* creating web/templates/post/show.html.eex
+* creating web/views/post_view.ex
+* creating test/controllers/post_controller_test.exs
+```
+
+It will tell us we need to add a line to our router file but since we skipped model it won't mention anything about ecto.migrate.
```

Comment:
I think we need just a couple of tiny changes here.
- add a comma before "but"
- add "the" before "model"
- add a comma after "model"
- add backticks around "ecto.migrate"

". . . , but since we skipped the model, . . . `ecto.migrate`."